pom.xml

```
@@ -604,6 +603,12 @@
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-compress</artifactId>
                 <version>${version.apache.commons.compress}</version>
+            </dependency>
+			<dependency>
```

Incorrectly aligned.

---

Hopefully I got that fixed. The alignment looked correct in my environment though so I fiddled with it to get it to be correct (hopefully).

---

Well, (unless you haven't committed + pushed), then -- no, it's not fixed. Have a look at this diff:
```suggestion
		<dependency>
```

Could you please keep a closer look at the diffs when you push? (They are always displayed here and you can easily spot such cosmetic things. Thanks! :)

---

I have no idea why its coming up looking like this i've worked on realigning a few times now. The pom is set how you want it to look on my PC so I have no current idea why its being pushed to look like this.